client/my-sites/themes/auto-loading-homepage-modal.jsx

```diff
@@ -187,7 +187,7 @@ class AutoLoadingHomepageModal extends Component {
 							checked={ 'use_new_homepage' === this.state.homepageAction }
 							onChange={ this.handleHomepageAction }
 							label={ translate(
-								"Use %(themeName)s's homepage content and make my existing homepage a draft.",
+								"Use %(themeName)s's homepage for my site and make my existing homepage a draft.",
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.